apps/api/src/drizzle/transaction.interceptor.ts

```diff
@@ -26,7 +30,10 @@ export class TransactionInterceptor implements NestInterceptor {
         const gqlContext = GqlExecutionContext.create(context);
         const ctx = gqlContext.getContext();
 
-        const db = drizzle(this.pool, { schema });
+        const db = drizzle(this.pool, {
```

Comment:
Why the db is not injected?

Reply:
Ya 3ammena, separate issue wallah :sob: See #2